code/modules/reagents/Chemistry-Goon-Pyrotechnics.dm

```diff
@@ -29,6 +29,7 @@
 	var/turf/simulated/T = get_turf(holder.my_atom)
```

Comment:
yeah, was gonna make it adjust fire stacks but then it becomes hellwater/burning spore toxin 2.0